index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio - Theophilus Doku</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <!-- Header -->
+    <header class="header-wrapper">
+        <a href="#" class="logo">
+            <h1>Sozo</h1>
+        </a>
+        <button class="menu_btn" type="button">
+            <img src="images/menu.png" alt="menu-button">
+        </button>
+    </header>
```

Comment:
- [ ] Since this toolbar is in charge of the navigation, it would be wise to wrap it inside a `<nav>` tag, so assistive technologies such as screen readers could easily identify what's this section about, improving tremendously the accessibility of your page! 🤓 🛠️  

```suggestion
    <header class="header-wrapper">
        <nav>
            <a href="#" class="logo">
                <h1>Sozo</h1>
            </a>
            <button class="menu_btn" type="button">
                <img src="images/menu.png" alt="menu-button">
            </button>
        </nav>
    </header>
```